setup.py

```diff
@@ -23,36 +25,38 @@
         "Environment :: Web Environment",
         "Framework :: Plone",
         "Framework :: Plone :: 5.2",
+        "Framework :: Plone :: 6.0",
         "Programming Language :: Python",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Operating System :: OS Independent",
         "License :: OSI Approved :: GNU General Public License v2 (GPLv2)",
     ],
-    keywords='Python Plone',
-    author='kitconcept GmbH',
-    author_email='info@kitconcept.com',
-    url='https://pypi.python.org/pypi/kitconcept.seo',
-    license='GPL version 2',
-    packages=find_packages('src', exclude=['ez_setup']),
-    namespace_packages=['kitconcept'],
-    package_dir={'': 'src'},
+    keywords="Python Plone",
+    author="kitconcept GmbH",
+    author_email="info@kitconcept.com",
+    url="https://pypi.python.org/pypi/kitconcept.seo",
+    license="GPL version 2",
+    packages=find_packages("src", exclude=["ez_setup"]),
+    namespace_packages=["kitconcept"],
+    package_dir={"": "src"},
     include_package_data=True,
     zip_safe=False,
+    python_requires=">=3.7",
     install_requires=[
-        'plone.api',
-        'Products.GenericSetup',
-        'setuptools',
-        'z3c.jbot',
-        'plone.behavior',
+        "plone.api",
```

Comment:
Just add Plone here, remove GS, plone.behavior and plone.api

Reply:
Don't we want to be explicit about the dependencies? I usually try to list every package that is imported. I remember that we tried to check this with CI. There is no way, as far as I can tell that we could automate a check if we are not explicit here.